platform/innovium/python-saithrift.mk

```diff
@@ -0,0 +1,6 @@
+# python-saithrift package
+
+PYTHON_SAITHRIFT_INVM = python-saithrift_1.2.1_amd64.deb
```

Comment:
amd64 -> $(CONFIGURED_ARCH)

Reply:
introduced in https://github.com/Azure/sonic-buildimage/pull/2980

Reply:
fixed